core/constants.py

```diff
@@ -59,6 +59,32 @@ def remove_comments(text: str) -> str:
     return re.sub(r'  //.*\n', r'', text)
 
 
+def get_package_file_contents(package: str, filepath: str) -> str:
```

Comment:
Why is this "generic" function in constants.py? If it is in this file I would expect it to only return the file contents of the constants file.

Reply:
https://github.com/oppia/oppia/pull/15078#discussion_r824218391 Please check this discussion from above. I notice I forgot to add a comment explaining why its put there, thanks I have added it back.
Any suggestion for where I might put this function?